src/server/symbols.map

```diff
@@ -915,6 +909,14 @@ MIR_SERVER_2.8 {
     vtable?for?mir::shell::WindowManagerTools;
     vtable?for?mir::time::Alarm;
     vtable?for?mir::time::AlarmFactory;
+    mir::DefaultServerConfiguration::the_keyboard_observer_registrar*;
+    mir::DefaultServerConfiguration::the_keyboard_observer_registrar*;
+    mir::shell::SurfaceReadyRegistrar::?SurfaceReadyRegistrar*;
+    mir::shell::SurfaceReadyRegistrar::SurfaceReadyRegistrar*;
+    typeinfo?for?mir::shell::SurfaceReadyRegistrar;
+    vtable?for?mir::shell::SurfaceReadyRegistrar;
+    mir::shell::SurfaceReadyRegistrar::register_surface*;
```

Comment:
Mir 2.8 has released; this is an ABI break, so everything should be in a shiny new `MIR_SERVER_2.9` stanza, right?